examples/server/README.md

```diff
@@ -42,6 +42,7 @@ see https://github.com/ggerganov/llama.cpp/issues/1437
 - `-to N`, `--timeout N`: Server read/write timeout in seconds. Default `600`.
 - `--host`: Set the hostname or ip address to listen. Default `127.0.0.1`.
 - `--port`: Set the port to listen. Default: `8080`.
+- `--public-domain`: Set a public domain which will be allowed for Cross Origin Requests. If you are using the server as an API from a browser, this is required.
```

Comment:
Please rename it to reference CORS at the very least, public domain might confuse people something like `--cors-allowed-origin`.

```
- `--cors-origin`: Set what origin (example.com) is allowed to access the API. Use * to allow all origins (insecure).  If you are using the server as an API from a browser, this parameter is required.
```

Reply:
++ and the user is sending cross origin requests.

Reply:
also, to be coherent with other parameters,  I suggest `--http-cors-origin`

Reply:
My thought was that a typical user probably wouldn't know what CORS is or why they need it. I've updated the flag to be "--http-cors-origin". There's potentially some consideration to avoiding "http" as the origin "https://example.org" is different from "http://example.org".